src/components/Modal/ConnectWallet.svelte

```diff
@@ -52,10 +52,7 @@
 
     <div slot="subtitle">
       <div class="text-small">
-        WalletConnect<br/>
-        <a href="https://walletconnect.com/registry?type=wallet" class="link">
-          View compatible wallets
-        </a>
+      Scan QR code with a <br />WalletConnect-compatible wallet
```

Comment:
Can we just leave this as `WalletConnect` like before and remove the link?

Reply:
Yeah sure 👍

Reply:
It seemed that `WalletConnect` by itself, was standing pretty alone so I thought an explanation was maybe better than to just show `WalletConnect`. 

Or maybe we could change the `Connect your wallet` to something alluding to `WalletConnect`?
Maybe `Connect with WalletConnect`?

Reply:
Maybe even better we could do a listing of different wallets, similar to Workstreams
<img width="300" alt="Bildschirmfoto 2022-08-12 um 14 33 57" src="https://user-images.githubusercontent.com/7912302/184354852-aec3c91a-92d1-49f1-b0e2-02c281a038ca.png">

The Metamask button would be like clicking on `Connect with Metamask` and when clicking on WalletConnect, we could implement into our own style, the WalletConnect QR Modal to allow connection with deep linking on mobile ref #287

Would probably need some rework on how we manage the wallet connect state inside the app

wdyt @cloudhead

Reply:
Two clicks vs. 1

Reply:
> Two clicks vs. 1

Okay 👌